### ISSUE TITLE
How do I negate a rule?

### ISSUE THREAD

**Issue by StephanBijzitter** [benmosher/eslint-plugin-import#1013]
For example this rule: `import/group-exports` (https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/group-exports.md)

I would like to do the exact opposite: forbid grouped named exports. Is that possible?

**Comment by ljharb** [comment]
No, if a rule does not explicitly support the inverse, then it's not possible to warn on it.

This seems like a potential enhancement to the rule, however.

**Comment by dfreeman** [comment]
@ljharb if we were to propose this as an enhancement to `group-exports`, any thoughts on a good way to specify that config for the rule? Maybe `{ preferInlineExports: true }` or something along those lines?

**Comment by ljharb** [comment]
I'd probably do something like a string option, either "enforce" (the default) or "forbid".

**Comment by beltekylevi** [comment]
@StephanBijzitter have you found a solution for this problem?

**Comment by ljharb** [comment]
@beltekylevi the solution is someone submitting a PR.

**Comment by zachmayer** [comment]
@StephanBijzitter did you ever find a way to encourage inline exports?

**Comment by zachmayer** [comment]
@dfreeman `{ preferInlineExports: true }` yes!  I can try my hand at making a PR here.
